### ISSUE TITLE
Unable to parse valid NACHA files containing POP and ARC SEC Codes when SCC is set to 200

### ISSUE THREAD

**Issue by vishwasbabu** [wadearnold/ach#1072]
ACH Version: `v1.18.6`

**What were you trying to do?**
Trying to parse valid NACHA files we received from a correspondent bank. The NACHA files themselves were very similar to the MOOV examples for https://github.com/moov-io/ach/blob/master/test/ach-arc-read/arc-debit.ach and https://github.com/moov-io/ach/blob/master/test/ach-pop-read/pop-debit.ach with one change, the Service Class Code being set to 200

**What did you expect to see?**
The files being parsed successfully

**What did you see?**
A validation error stating

```
{
    "ID": "b9c09fab3229f13f8cdadd7d0fea1c50bb71de4d",
    "error": "line:4 record:Batches *ach.BatchError batch #1 (POP) ServiceClassCode header SCC is not valid for this batch's type: 220"
}
```

**How can we reproduce the problem?**
Replace the SCC on https://github.com/moov-io/ach/blob/master/test/ach-pop-read/pop-debit.ach with 200 and try parsing the file

**Comment by vishwasbabu** [comment]
While it is true that Ideally the batches for POP and ARC should have had Service Class Code set to 225 / Debits Only, It does look like there are ACH Originators who set the Service Class Code for a batch to Mixed Debits and Credits (200) and send out only Debits. Maybe we should modify the strict validation the Moov library does at https://github.com/moov-io/ach/blob/master/batchPOP.go#L65 to allow the same?

**Comment by nazeer1100126** [comment]
I have sent MR.

**Comment by adamdecaf** [comment]
I've released v1.19.2 with this fix. Thanks again! 

https://github.com/moov-io/ach/releases/tag/v1.19.2
